src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1562,5 +1563,19 @@ public Location get(LootGenerateEvent event) {
 				}
 			}, EventValues.TIME_NOW);
 		}
+
+		// InventoryItemMoveEvent - does not extend InventoryEvent
```

Comment:
Does it support `event-item`? as the event does have getItem method

Reply:
iirc the event value was registered beforehand and event-item works as expected. It was midnight while I made this so I will re-verify it to make sure!

Reply:
Whoops, it wasn't there. Confused but now resolved!